benches/containerd-shim-benchmarks/benches/wasi-demo-app-benchmarks.rs

```diff
@@ -0,0 +1,62 @@
+use std::process::Command;
+use std::time::{Duration, Instant};
+use criterion::{criterion_group, criterion_main, Criterion};
+
+fn run_container(runtime: &str) -> Duration {
+    let start = Instant::now();
+    
+    let output = Command::new("sudo")
+        .args([
+            "ctr",
+            "run",
+            "--rm",
+            &format!("--runtime=io.containerd.{}.v1", runtime),
+            "ghcr.io/containerd/runwasi/wasi-demo-app:latest",
```

Comment:
could we do one with the artifact image too?  It would be interesting to see the difference if any.

Reply:
Just added it